src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -44,6 +44,8 @@ internal static class WellKnownTypeNames
         public const string MicrosoftCodeAnalysisDiagnosticsCompilationStartAnalysisContext = "Microsoft.CodeAnalysis.Diagnostics.CompilationStartAnalysisContext";
         public const string MicrosoftCodeAnalysisDiagnosticsDiagnosticAnalyzer = "Microsoft.CodeAnalysis.Diagnostics.DiagnosticAnalyzer";
         public const string MicrosoftCodeAnalysisDiagnosticsDiagnosticAnalyzerAttribute = "Microsoft.CodeAnalysis.Diagnostics.DiagnosticAnalyzerAttribute";
+        public const string MicrosoftCodeAnalysisISourceGenerator = "Microsoft.CodeAnalysis.ISourceGenerator";
+        public const string MicrosoftCodeAnalysisGeneratorAttribute = "Microsoft.CodeAnalysis.GeneratorAttribute";
```

Comment:
The file was intended to be sorted alphabetically.

https://github.com/dotnet/roslyn-analyzers/blob/d1fbafacc364f3c32168478df0b060b6f541d4c8/src/Utilities/Compiler/WellKnownTypeNames.cs#L5

Reply:
Next analyzer... `WellKnownTypesIsSortedAnalyzer`